Makefile

```diff
@@ -81,3 +81,9 @@ py-lint: pre-commit
 
 js-format:
 	cd superset-frontend; npm run prettier
+
+flask-app:
+	flask run -p 8088 --with-threads --reload --debugger
+
+node-app:
+	cd superset-frontend; npm run dev-server
```

Comment:
And here.

Reply:
Running the command you still land in the root just tested it locally so we don't need to change it